### PR TITLE
minor corrections and edits to vignettes

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: vignettes/rowwise.Rmd [hunk 1]
```diff
@@ -274,7 +274,7 @@ df <- tribble(
   ~ n, ~ min, ~ max,
     1,     0,     1,
     2,    10,   100,
-    3,   100,  1000,
+    3,   100,  1000
```
Comment (krlmlr) [inline vignettes/rowwise.Rmd:277]: The trailing comma here is explicitly supported, and allows adding more rows without editing the previous line.

File: vignettes/two-table.Rmd [hunk 2]
```diff
@@ -71,8 +71,7 @@ As well as `x` and `y`, each mutating join takes an argument `by` that controls
     Note that the year columns in the output are disambiguated with a suffix.
 
   * A named character vector: `by = c("x" = "a")`. This will
-    match variable `x` in table `x` to variable `a` in table `y`. The 
-    variables from use will be used in the output.
+    match variable `x` in table `x` to variable `a` in table `y`.
```
Comment (krlmlr) [inline vignettes/two-table.Rmd:74]: "variables from use" might refer to the `by` argument -- was it called `use` before?

**Review by krlmlr — COMMENTED** [review]
Thanks! This feels like it could be three pull requests: typos, suggestions, and new contents. Would you like to file a separate pull request that covers only the typos?

**Comment by IndrajeetPatil** [comment]
> This feels like it could be three pull requests: typos, suggestions, and new contents.

Thanks for the feedback!
Sure, I can break this PR into three separate PRs.

**Pull request closed without merging**